src/java.base/share/classes/java/lang/invoke/InvokerBytecodeGenerator.java

```diff
@@ -96,14 +94,11 @@ class InvokerBytecodeGenerator {
     private int[]       localsMap;    // index
     private Class<?>[]  localClasses; // type
 
-    /** ASM bytecode generation. */
-    private ClassWriter cw;
-    private MethodVisitor mv;
     private final List<ClassData> classData = new ArrayList<>();
 
     /** Single element internal class name lookup cache. */
     private Class<?> lastClass;
-    private String lastInternalName;
+    private ClassDesc lastInternalName;
```

Comment:
`s/lastInternalName/lastClassDesc/` as it's no longer an internal name.

The comment in line 99 should also be updated: "Single element class descriptor lookup cache"

Reply:
fixed, thanks.